src/rdkafka_metadata_cache.c

```diff
@@ -194,7 +194,7 @@ int rd_kafka_metadata_partition_id_cmp (const void *_a,
  *
  * This makes a copy of \p topic
  *
- * @locks rd_kafka_wrlock()
+ * @locks_required rd_kafka_wrlock()
```

Comment:
it would probably be worth doing a PR that changed the terminology `@locks` to `@locks_required` everywhere. simple and makes it a little bit more readable.